src/components/MyArea/MyArea.constants.tsx

```diff
@@ -0,0 +1,2 @@
+export const MAP_URL: string =
+  'https://data.amsterdam.nl/data/?modus=kaart&achtergrond=topo_rd_zw&center=52.3717228%2C4.8927377&embed=true&legenda=true&zoom=10';
```

Comment:
Is this a default center on the "middle" of Amsterdam?

Reply:
Yes